apps/site/assets/ts/schedule/components/LineDiagram.tsx

```diff
@@ -1,12 +1,26 @@
 import React, { ReactElement } from "react";
 import { LineDiagramStop, StopData } from "./__schedule";
 import { modeIcon, accessibleIcon, parkingIcon } from "../../helpers/icon";
-import { Route } from "../../__v3api";
+import { Alert, Route } from "../../__v3api";
 
 interface Props {
   lineDiagram: LineDiagramStop[];
 }
 
+const maybeAlert = (alerts: Alert[]): ReactElement<HTMLElement> | null => {
+  const highPriorityAlerts = alerts.filter(alert => alert.priority === "high");
+  if (highPriorityAlerts.length === 0) return null;
+  return (
+    <i
```

Comment:
I know this is styling which you said is to come in another pass, but can we start out with the alert svg icon instead of using font icons? There's already a helper for it. https://github.com/mbta/dotcom/blob/3ceb65e08aa2beef103db8df8e8b67354a91816f/apps/site/assets/ts/helpers/icon.tsx#L23
And leave out the tooltip for now, add a sibling "sr-only" label of "service alert or delay".

Reply:
What about using `renderSVG("c-svg__icon-alerts-triangle", alertIcon)`?

Reply:
That would produce the same thing as `alertIcon("c-svg__icon-alerts-triangle")`. I think renderSVG came before the extra icon helpers, it would be easier to change all of the alert icons at once with alertIcon(className)...but we already have exceptions so it doesn't matter.

https://github.com/mbta/dotcom/blob/3ceb65e08aa2beef103db8df8e8b67354a91816f/apps/site/assets/ts/helpers/icon.tsx#L23-L24